dev/src/codewind/project/ProjectType.ts

```diff
@@ -127,7 +127,11 @@ export class ProjectType {
             ProjectType.InternalTypes.MICROPROFILE,
             ProjectType.InternalTypes.NODE,
             ProjectType.InternalTypes.SPRING
-        ].includes(this.internalType);
+        ].includes(this.internalType) ||
+            ([ProjectType.InternalTypes.DOCKER].includes(this.internalType) &&
+            [ProjectType.Languages.JAVA].map((lang) => lang.toString().toLowerCase())
+            .includes(this.language.toLowerCase())
```

Comment:
More readable would be

```
||
this.internalType === ProjectType.InternalTypes.DOCKER && this.language.toLowerCase === ProjectType.Languages.JAVA

Reply:
Sold! https://github.com/eclipse/codewind-vscode/pull/425/commits/7e3ceb7bcc5fcff12baa2427adf87c4b9fc70c2d